packages/stylelint-config/index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 module.exports = {
-	extends: 'stylelint-config-recommended',
+	extends: [ 'stylelint-config-recommended' ].map( require.resolve ),
```

Comment:
What's the reason for using `map` here` with a single item?

Reply:
I made it an array when I was previously using `@stylistic/stylelint-config`, but when I removed that config, I kept it as an array.

There was then a second error where it wouldn't find the config, so I fixed that with map/require.resolve as that's how the other configs are set up.

I can convert this to a non-mapped version if you'd prefer that?

Reply:
I see it is used in other places with multiple items. It's fine.